pyproject.toml

```diff
@@ -11,7 +11,7 @@ license = { text = "VSL" }
 dynamic = ["version"]
 
 dependencies = [
-    "pefile>=2024.8.26",
```

Comment:
This is the wrong way to fix the issue.  I don't think pefile is at fault, I believe it's the way that pyinstaller used it, and pyinstaller is only used by devs to construct an exe (and it's only a performance degradation), so I think we bump pyinstaller down (it used to start at 6.5, but during the toml update it got bump to the latest version at that time).